### PR TITLE
Cleanup of README

### PR DESCRIPTION
**Pull request by arves100**
- Added Windows build instructions (vcpkg+choco)
- Removed old library requirements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,19 +9,23 @@
 
 ### Install Prerequisites
 #### Install Git
- * If you want to use Git in the command line, run `sudo apt install git`
+ * If you want to use Git in the command line, run `sudo apt install git`, you can get [Git for Windows](https://git-scm.com/) or use `choco install git`
```
Comment (UndarkAido) [inline README.md:12]: Clarify platforms

Comment (UndarkAido) [reply README.md:12]: Something like "run `sudo apt install git` on Ubuntu or on Windows download git from XX or run XX in the command line if you have chocolatey"

File: README.md [hunk 2]
```diff
@@ -9,19 +9,23 @@
...
  * If you're less terminal-inclined, you can install a Git GUI wrapper. [There's a nice list here.](https://git-scm.com/downloads/guis/)
     * I would personally recommend [GitKraken](https://www.gitkraken.com/). The free version has everyhting you need, (although their conflict resolver for Pro is nice,) and you can also get Pro free with [GitHub Education](https://education.github.com/)!
 #### <sup>AND</sup> Install C++
  1. Install a C++ compiler
     * the typical standard is `gcc`, you can install it with `sudo apt install gcc`
     * `clang` is an alternative that I feel has better error statements, you can install it with `sudo apt install clang`
- 2. Install CMake with `sudo apt install cmake`
+    * The typical standard compiler for Windows is [Microsoft Visual C++](https://www.visualstudio.com/)
+    * [MinGW](https://www.msys2.org/) or [Cygwin](https://www.cygwin.com/) are valid alternatives if you like GCC, oh there's also [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10) as well ^^.
```
Comment (UndarkAido) [inline README.md:20]: It isn't clear this line applies to Windows

File: README.md [hunk 3]
```diff
@@ -9,19 +9,23 @@
...
+	
+ 2. Install CMake with `sudo apt install cmake` or by downloading it from [CMake](https://cmake.org/download/) or get it with `choco install cmake`.
```
Comment (UndarkAido) [inline README.md:22]: Clarify platforms

File: README.md [hunk 4]
```diff
@@ -9,19 +9,23 @@
...
 #### <sup>AND</sup> Install Libraries
- * Debian-based (Ubuntu) `sudo apt install build-essential libboost-all-dev libssl-dev libcrypto++-dev libcurl4-openssl-dev`
- * Fedora/RHEL-based `sudo dnf install openssl-devel boost-devel cryptopp-devel cryptopp`
+ * Debian-based (Ubuntu) `sudo apt install build-essential libboost-all-dev libssl-dev`
+ * Fedora/RHEL-based `sudo dnf install openssl-devel boost-devel`
+ * Windows Get [vcpkg](https://github.com/microsoft/vcpkg), follow the page instruction to install it, then type`vcpkg install boost-system boost-beast openssl`
```
Comment (UndarkAido) [inline README.md:26]: Grammar here feels awkward. Maybe we need a `tools` section for things like `git`, `vcpkg`, `cmake`, and `choco`?

File: README.md [hunk 5]
```diff
@@ -30,7 +34,9 @@
  * Click on your new 'Settings' tab, and rename your new repo
  
 #### Clone the Code
-Run `git clone --recursive https://github.com/discordpp/echo-bot.git` in your desired directory. If you made a fork, replace `discordpp` and `echo-bot` as applicable.
+Run `git clone --recursive https://github.com/discordpp/echo-bot.git` in your desired directory.
+Make sure to run `git submodule update --init --recursive` in the new echo-bot directory to make sure it downloads all submodules.
```
Comment (UndarkAido) [inline README.md:38]: `git submodule update --init --recursive` is not required if you `clone` with `--recursive`

File: README.md [hunk 6]
```diff
@@ -55,8 +61,8 @@ Run `git clone --recursive https://github.com/discordpp/echo-bot.git` in your de
  1. Create a `build` directory (folder) with `mkdir build`
  2. Enter the folder with `cd build`
  3. Go back to your Application's 'bot' page from before, and copy the token. Run `echo Bot [token] > token.dat`, replacing the brackets and the word token
- 4. Build the makefiles with `cmake ../`
- 5. Build the bot with `make`
+ 4. Build the makefiles with `cmake ../` (If you are using vcpkg then `cmake -DCMAKE_TOOLCHAIN_FILE="your vcpkg.cmake folder" ../`)
+ 5. Build the bot with `make` (if you are using Visual Studio just open `echo_bot.sln` and build from there)
```
Comment (UndarkAido) [inline README.md:65]: Remove "just"

File: README.md (hunk 4 above)
Comment (UndarkAido) [reply README.md:26]: Oh, and maybe something about how `vcpkg` is cross-platform?

File: README.md [hunk 7]
```diff
@@ -55,8 +61,8 @@ Run `git clone --recursive https://github.com/discordpp/echo-bot.git` in your de
  1. Create a `build` directory (folder) with `mkdir build`
  2. Enter the folder with `cd build`
  3. Go back to your Application's 'bot' page from before, and copy the token. Run `echo Bot [token] > token.dat`, replacing the brackets and the word token
- 4. Build the makefiles with `cmake ../`
- 5. Build the bot with `make`
+ 4. Build the makefiles with `cmake ../` (If you are using vcpkg then `cmake -DCMAKE_TOOLCHAIN_FILE="your vcpkg.cmake folder" ../`)
```
Comment (UndarkAido) [inline README.md:64]: Maybe add the `vcpkg` stuff as an indented subpoint of this one instead of in parenthesis?

File: README.md (hunk 4 above)
Comment (arves100) [reply README.md:26]: Should I just point vcpkg in all platforms or say something like you can use vcpkg on linux/mac as well?

**Comment by arves100** [comment]
Ok, I've pushed a new commit with the new README.

- Moved vcpkg cmake command into a subpoint
- Removed git submodule update
- Moved cmake, vcpkg istructions into the Prerequisites
- I guess I explained what vcpkg is and that is required for Windows, and how to get it
- Updated the compiler to clarify the platforms

Let me know if there's anything else I need to modify.

File: README.md [hunk 8]
```diff
@@ -9,19 +9,28 @@
...
+ * If you want to use Git in the command line, run `sudo apt install git` on Ubuntu, on Windows you can download [Git for Windows](https://git-scm.com/) or use `choco install git` if you have Chocolatey installed.
  * If you're less terminal-inclined, you can install a Git GUI wrapper. [There's a nice list here.](https://git-scm.com/downloads/guis/)
     * I would personally recommend [GitKraken](https://www.gitkraken.com/). The free version has everyhting you need, (although their conflict resolver for Pro is nice,) and you can also get Pro free with [GitHub Education](https://education.github.com/)!
+#### Install CMake
+ * Debian-based (Ubuntu) run `sudo apt install cmake`
+ * Fedora/RHEL-based `sudo dnf install cmake`
+ * Windows Download [CMake](https://cmake.org/download/) or run `choco install cmake` if you have Chocolatey installed.
+#### Install vcpkg
+ * Vcpkg is a cross platform library manager, you can install it by reading the instructions on this [page](https://github.com/microsoft/vcpkg)
+ * This program is required to build echo-bot under Windows.
```
Comment (UndarkAido) [inline README.md:21]: Refer to Echo with the proper capitalized name at the top of the repo, not the constriined-formatting repo name

File: README.md [hunk 9]
```diff
@@ -9,19 +9,28 @@
...
 #### <sup>AND</sup> Install C++
  1. Install a C++ compiler
-    * the typical standard is `gcc`, you can install it with `sudo apt install gcc`
-    * `clang` is an alternative that I feel has better error statements, you can install it with `sudo apt install clang`
- 2. Install CMake with `sudo apt install cmake`
+    * the typical standard on Linux is `gcc`, you can install it on Ubuntu with `sudo apt install gcc`
+    * `clang` is an alternative that I feel has better error statements, you can install it on Ubuntu with `sudo apt install clang`
+    * The typical compiler on Windows is [Microsoft Visual C++](https://www.visualstudio.com/)
+    * [MinGW](https://www.msys2.org/) or [Cygwin](https://www.cygwin.com/) are valid alternatives if you like GCC running on Windows, oh there's also [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10) as well ^^.
```
Comment (UndarkAido) [inline README.md:27]: Maybe clarify a tiny bit what WSL does? Something like "WSL, which allows you to use a Linux development environment on Windows"

Comment (UndarkAido) [inline README.md:27]: Replace 'oh' with 'and'?

File: README.md [hunk 10]
```diff
@@ -9,19 +9,28 @@
 
 ### Install Prerequisites
 #### Install Git
- * If you want to use Git in the command line, run `sudo apt install git`
+ * If you want to use Git in the command line, run `sudo apt install git` on Ubuntu, on Windows you can download [Git for Windows](https://git-scm.com/) or use `choco install git` if you have Chocolatey installed.
  * If you're less terminal-inclined, you can install a Git GUI wrapper. [There's a nice list here.](https://git-scm.com/downloads/guis/)
     * I would personally recommend [GitKraken](https://www.gitkraken.com/). The free version has everyhting you need, (although their conflict resolver for Pro is nice,) and you can also get Pro free with [GitHub Education](https://education.github.com/)!
+#### Install CMake
```
Comment (UndarkAido) [inline README.md:15]: Include <sup>AND</sup>

File: README.md [hunk 11]
```diff
@@ -9,19 +9,28 @@
...
+ * Debian-based (Ubuntu) run `sudo apt install cmake`
+ * Fedora/RHEL-based `sudo dnf install cmake`
+ * Windows Download [CMake](https://cmake.org/download/) or run `choco install cmake` if you have Chocolatey installed.
+#### Install vcpkg
```
Comment (UndarkAido) [inline README.md:19]: Include something like <sup>OPTIONALLY</sup>

File: README.md [hunk 12]
```diff
@@ -30,7 +39,8 @@
  * Click on your new 'Settings' tab, and rename your new repo
  
 #### Clone the Code
-Run `git clone --recursive https://github.com/discordpp/echo-bot.git` in your desired directory. If you made a fork, replace `discordpp` and `echo-bot` as applicable.
+Run `git clone --recursive https://github.com/discordpp/echo-bot.git` in your desired directory.
+If you made a fork, replace `discordpp` and `echo-bot` as applicable.
```
Comment (UndarkAido) [inline README.md:43]: Why replace `discordpp`?

File: README.md [hunk 13]
```diff
@@ -9,19 +9,28 @@
...
+    * [MinGW](https://www.msys2.org/) or [Cygwin](https://www.cygwin.com/) are valid alternatives if you like GCC running on Windows, oh there's also [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10) as well ^^.
 #### <sup>AND</sup> Install Libraries
- * Debian-based (Ubuntu) `sudo apt install build-essential libboost-all-dev libssl-dev libcrypto++-dev libcurl4-openssl-dev`
- * Fedora/RHEL-based `sudo dnf install openssl-devel boost-devel cryptopp-devel cryptopp`
+ * Debian-based (Ubuntu) `sudo apt install build-essential libboost-all-dev libssl-dev`
+ * Fedora/RHEL-based `sudo dnf install openssl-devel boost-devel`
+ * Vcpkg `vcpkg install boost-system boost-beast openssl`
 #### <sup>OR</sup> Install Everything at Once
-`sudo apt install git clang cmake build-essential libboost-all-dev libssl-dev libcrypto++-dev libcurl4-openssl-dev`
+ * Debian-based `sudo apt install git clang cmake build-essential libboost-all-dev libssl-dev`
```
Comment (UndarkAido) [inline README.md:33]: Add (Ubuntu) like above

File: README.md (hunk 4 above)
Comment (UndarkAido) [reply README.md]: That would be good

File: README.md [hunk 14]
```diff
@@ -8,29 +8,29 @@
 ## How to Get Started <sup>Let's get it sta-arted! Let's get it started in here!</sup>
 
 ### Install Prerequisites
-#### Install Git
+#### <sup>AND</sup>Install Git
```
Comment (UndarkAido) [inline README.md:11]: It doesn't make sense for the first point to have an 'AND'

**Comment by arves100** [comment]
Updated.

**Pull request merged**